Documentation/CLionConfiguration.md

```diff
@@ -2,13 +2,24 @@
 
 CLion can integrate with CMake to provide code comprehension features.
 
-After opening the `serenity` repository in CLion as a new project, go to "`File->Settings->Build, Execution, Deployment->Cmake`",
-and set the following fields: (Assuming you use `Ninja` as the build system and configured the CMake build directory to `Build/i686`)
+After opening the `serenity` repository in CLion as a new project, the "`Open Project Wizard`" window will open, from here set the following fields:
+
+(Assuming you use `Ninja` as the build system and configured the CMake build directory to `Build/i686`)
+
+`Build type`: `Default`
+
+> _CMake will complain with  any other build type, make sure to use `Default` so that `CMAKE_BUILD_TYPE` is empty in the `Build Directory/CMakeCache.txt` file._
```

Comment:
Just a little above you assumed the build dir is `Build/i686`, so `Build Directory/CMakeCache.txt` is a little odd (also, I'd normally expect something like `$BUILD_DIRECTORY/CMakeCache.txt` for paths with placeholders.

Also, extraneous space here: 
```
> complain with  any other
               ^
```

Reply:
Thanks for reviewing!
Fixed both things in newest commit.